dev/all_tests.yaml

```diff
@@ -270,7 +270,6 @@ tests:
       services: [clouddriver, front50, gate, orca, echo]
     quota:
       appengine_deployment: 1
-    path: citest/tests/gcs_pubsub_gae_test.py
```

Comment:
Does this need replaced with something?

Reply:
Nope, since the test name & entry above match